app/assets/stylesheets/top_bar_styling.scss

```diff
@@ -0,0 +1,20 @@
+// #background {
+//  background: url(/assets/Boston_Skyline_Night_BW.png) no-repeat fixed center;
+//  background-size: contain;
+//  background-repeat: no-repeat;
+//  width: 100%;
+//  height: 100%;
+//  padding-top: 59.05%;
+//}
```

Comment:
It's going to be used in the next branch, we just changed the order of things we were styling and wanted to have them reviewed in chunks